### ISSUE TITLE
Not enough information, lessons dont provide too much help

### ISSUE THREAD

**Issue by Myrza711** [learn-co-curriculum/phase-0-css-kitten-lab#4]
### Canvas Link

https://learning.flatironschool.com/courses/5055/assignments/152491?module_item_id=324529

### Concern

All the lessons are really hard to follow. The lessons dont explain at all how to code correctly. You provide us only with do this and do that. Wish you could provide us with the right codes but we must decide where to put them. This is only a pre-work we are not suppose to know all, since everyone comes with no IT background. And doing it online and asking help in the Slack doestn help either. I could be sitting for days trying to figure out the problem with one assignment. Sometimes even Google doesnt help. I can only imagine when the actual course starts, how hard its going to be.

### Suggested Changes

Provide us with the correct codes, and we figure out where to code them. Not just, <you must update the header> or <Select the image that has an attribute value matching "Kitty 5" (using attribute selector)> This is really hard to know because in the lesson prior you didnt really provide us with the right codes. And each lesson should have the right codes on the side. We are suppose to learn not figure out how to do this from the start.

**Comment by graciemcguire** [comment]
Hi @Myrza711 ! 
Thank you for your feedback, we really appreciate it. We are constantly updating and modifying the curriculum and will take your feedback into consideration!

In the mean time, check out [this lesson](https://learning.flatironschool.com/courses/3280/pages/css-fundamentals?module_item_id=142673), it goes over what attributes and selectors are, how to find them, and how we can use them in our CSS. 

Remember, learning to code is hard, and it takes a lot of practice to learn this stuff, so good job! You've come really far already and you've put a ton of work in!

Thanks again for your feedback, and happy coding!
